Source/CombatExtended/CombatExtended/CritDefGenerator.cs

```diff
@@ -11,15 +11,15 @@ namespace CombatExtended
     public static class CritDefGenerator
     {
         public static IEnumerable<DamageDef> ImpliedCritDefs()
-        {
-            foreach(DamageDef current in from def in DefDatabase<DamageDef>.AllDefs where def.externalViolence select def)
+        {                                                                           //TODO: ExternalViolence for what?
```

Comment:
Crit defs are obsolete now that armor penetration can be passed via damage info. They only existed so sharp weapon crits could bypass armor. Should increase penetration value in Verb_MeleeAttackCE instead.

Reply:
Sounds like optimization, but I will leave it as is for now, since it should at least run before we remove redundant code. (In my opinion at least)